Models/Patch-reviews-model.js

```diff
@@ -0,0 +1,6 @@
+const db = require('../db/connection')
+
+exports.patchReviewsModel = (reviewID, voteChange) => {
+    return db.query(`UPDATE reviews SET votes = votes + $1 WHERE review_id=$2 RETURNING*;`, [voteChange, reviewID])
```

Comment:
You'll need a .then() block to deal with the return of the SQL query and how you want to send it back to the controller for .send() to deal with